### PR TITLE
Create Mapping Plot Classes

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
This PR addressing the mapping side of issue #26. 

Latitude, longitude and data are required for these plots, while other args are set to defaults. However, creating this as an object allows the user to be more declarative about how to plot their data.

### CODE REVIEW

File: src/emcpy/plots/map_plots.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+class MapScatter:
+
+    plottype = 'map_scatter'
```
Comment (CoryMartin-NOAA) [inline src/emcpy/plots/map_plots.py]: does this need a self?

**Review by CoryMartin-NOAA — COMMENTED** [review]
Question: `plottype`. Should that be `self.plottype`? Or are you wanting to call:
`print(MapContour.plottype)`?

File: src/emcpy/plots/map_plots.py (hunk 1 above)
Comment (aerorahul) [reply src/emcpy/plots/map_plots.py]: There is no `self` in the class attribute and you can access it as `MapScatter.plottype`
Not sure what is intended here.

Comment (aerorahul) [reply src/emcpy/plots/map_plots.py]: Most likely @kevindougherty-noaa is trying to treat this as a `global` variable in the class.

Comment (CoryMartin-NOAA) [reply src/emcpy/plots/map_plots.py]: Yeah I guess that's what I'm asking, I guess the question is when/how is `plottype` used down the road.

Comment (kevindougherty-noaa) [reply src/emcpy/plots/map_plots.py]: You know what, I think I confused myself reading class docs and that it doesn't really matter if it is a class variable or an instance variable. However, for it's purpose in the CreateMap class that I have in PR #29, it needs that variable to determine which function to call when plotting on the map.

Comment (aerorahul) [reply src/emcpy/plots/map_plots.py]: In that case why not set it in the `__init__` instead of making it an instance variable.

Comment (kevindougherty-noaa) [reply src/emcpy/plots/map_plots.py]: I'll revert it back to what I had in the first place.

**Review by aerorahul — APPROVED** [review]

**Pull request merged**